Contributions/Criteria-Documentation-Evaluation.md

```diff
@@ -0,0 +1,114 @@
+| Not Applicable | It does not make sense to use this criterion for this evaluation. |
+
+## Document review process:
+
+Entry Conditions:
+
+* An enumeration method (e.g. headings, line numbers, etc.) should be available for ease of review document feedback references.
+* A mechanism should be available for tracking changes to a review document.
+* Author requests the review
+  * Author can use the review criteria to determine if the document is mature enough to be reviewed
+  * The Author fills out the file metadata
+  * The Author fills out the file header metadata
+* Review lead is selected from the approved list
+* Reviewers are selected by the review lead
+* Relevant documents are made available
+  * Perform review
+    * Violations should be noted as: Critical (C), Severe (S), Minor (M), Observation (O), Question (Q), Not Applicable (N)
+  * Mark feedback according to criteria
+  * Feedback should be included as comments unless the change is minor
+  * Feedback should be assigned to the author to address
+  * Note to author:
+    * The reviewers are not directing their feedback at you or your ability.  The purpose of this is to find potential bugs and minimize systematic error in the documentation
+
+Exit Condition:
+
+* Fewer remaining major defects per page than the agreed exit standard
+* Recommendation is one (1) Severe (or higher) defect per page as an initial exit criterion
+
+## Review Checklist
+
+| Review ID | Criterion | Reference | Rating | Comments |
+| :---- | :---- | :---- | :---- | :---- |
+| ID\_01 | Is the document unambiguously clear to the target audience? | Location(s) of the violation of the criterion | C, S, M, O, Q, N  | Feedback on what the problem is and a suggested correction. |
+| ID\_02 | Do the stakeholders and consumers of the document understand its meaning? |  |  |  |
+| ID\_03 | Is there sufficient detail? |  |  |  |
+| ID\_04 | Are complex concepts described as measurable elementary concepts? |  |  |  |
+| ID\_05 | Is there a scale defined for any measurable concepts? |  |  |  |
+| ID\_06 | Are there benchmarks defined for any quantitative references? |  |  |  |
+| ID\_07 | Are there targets defined for any quantitative references? |  |  |  |
+| ID\_08 | If there are any performance ratings, does the document specify when those ratings are applicable? |  |  |  |
+| ID\_09 | Does the document sufficiently describe the capabilities and characteristics it is intended to cover? |  |  |  |
+| ID\_10 | Is the document accurate? |  |  |  |
+| ID\_11 | Is the document free from error? |  |  |  |
+| ID\_12 | Does the document contain any contradictions? |  |  |  |
+| ID\_13 | Can all of any assertions in the document be reproduced? |  |  |  |
+| ID\_14 | Are there any hypotheses in the document? |  |  |  |
+| ID\_15 | Does the document specify the project sufficiently such that it can be reproduced? |  |  |  |
```

Comment:
What do we mean by 'project' in this context? Perhaps this needs to be linked with the hypothesis item, requiring hypotheses to be supported by documented demonstrations specifying reproducible evidence?

Reply:
(It might also be helpful to make this and some of the other examples less abstract, by relating to our specific context. For example: "Claims or hypotheses about software must be backed by references to the associated source code or published documentation (ideally in the public domain), with specific, reproducible tests to demonstrate or confirm the claim."

Reply:
I added the "Claim or hypothesis" section to the definition of verifiable.